inference/worker/chat_chain.py

```diff
@@ -355,11 +379,17 @@ def handle_conversation(
             "language",
             "current_time",
             "action_input_format",
+            "custom_instructions",
         ] + (["tools_names"] if plugin_enabled else [])
 
         # TODO: Consider passing language from the UI here
         prompt_template = PromptTemplate(input_variables=input_variables, template=TEMPLATE)
 
+        custom_instructions = f"""\n{CUSTOM_INSTRUCTIONS_PREFIX.format(
```

Comment:
Also here - surely if `user_profile` and `user_response_instructions` are empty, we want this value `custom_instructions` to also be an entirely empty string, without the contents of `CUSTOM_INSTRUCTIONS_PREFIX`